NakedJavaScript/mensa-projekt
# M64]sql code auf injections prüfen

# What has changed:
* Add sanitizing function for the $_Get,$_Post forms, this should prevent MySQL injections from users.
# How to test:
* Go to any forms on our website and try to type in a valid MySQL statement, you should get an error message preventing an injection. Try this with all forms that we have.
